webcompat/static/js/lib/issues.js

```diff
@@ -434,19 +434,17 @@ issues.MainView = Backbone.View.extend({
         }, this)
       )
       .error(function(response) {
-        var msg;
-        if (
-          response &&
-          response.responseJSON &&
-          response.responseJSON.message === "API call. Not Found"
```

Comment:
Turns out this was broken. Caring about the `message`, rather than the `status` was a silly idea.